### PR TITLE
Upgrade to `three.js r104`

### PR DESCRIPTION
**Pull request by fabienrohrer**
Solves #455

- [x] Upgrade to `three.js r104`
- [x] Include `three.js` and `GPUPicker` to `webots.min.js`
- [x] Fix GPU picker to let it work with `three.js r104`
- [x] Fix `Khepera1` import
- [x] Fix ESLint issues in `gpu_picker.js`
- [x] Use ES6 features in `gpu_picker.js`
- [x] Upgrade doc dependencies

### CODE REVIEW

**Comment by fabienrohrer** [comment]
For the reviewer:

I temporary pushed `webots.min.upgrade-threejs.js` on the FTP and commit [these temporary modifications](https://github.com/omichel/webots/pull/463/commits/26b53c1c4a52ea787e37eece52618c3e08bc9231) to be able to test the local streaming and this url:
https://cyberbotics.com/doc/guide/aibo-ers7?version=upgrade-threejs

This commit should be rolled back and the FTP file should be overridden when merging this branch.

**Comment by fabienrohrer** [comment]
👍

**Comment by stefaniapedrazzi** [comment]
:eyes:

**Comment by omichel** [comment]
Why not simply using the animated GIF lady bug that we have in the current webots.js?

**Comment by fabienrohrer** [comment]
It's a good idea, let me open an issue about this.

File: resources/web/wwi/gpu_picker.js [hunk 1]
```diff
@@ -0,0 +1,483 @@
...
+    this.needUpdate = true;
+  }
+
+  pick(mouse, raycaster) {
+    this.update();
+    var index = mouse.x + (this.pickingTexture.height - mouse.y) * this.pickingTexture.width;
+    // interpret the pixel as an ID
+    var id = (this.pixelBuffer[index * 4 + 2] * 255 * 255) + (this.pixelBuffer[index * 4 + 1] * 255) + (this.pixelBuffer[index * 4 + 0]);
+    // get object with this id in range
+    // var object = this._getObject(id);
```
[the remark this answers is not in the document]
Comment (fabienrohrer) [reply resources/web/wwi/gpu_picker.js]: I prefer to remove this comment too. Quite useless..

**Comment by fabienrohrer** [comment]
👍

**Pull request merged**